app/assets/stylesheets/new_admin_layout.scss

```diff
@@ -51,6 +51,10 @@
   }
 }
 
+a.gem-c-button.govuk-button {
```

Comment:
Why do we need this?

Reply:
Otherwise the button text color is the link color. What's a good fix for that?

Reply:
Raised this PR https://github.com/alphagov/govuk_publishing_components/pull/582 to fix the issue in govuk_publishing_components. Once that's release I'll add a commit to this branch to remove this line and update the the new version of govuk_publishing_components.